clj/test/slipstream/ui/util/pattern_test.clj

```diff
@@ -22,7 +22,7 @@
 
 (defn- match-not-forbidden
   [matches? s]
-  (let [expected  (if matches? s nil)
+  (let [expected  (when matches? s)
```

Comment:
@loomis I want to understand this.  Why not simply `(if matches? s)` as it will return `nil` from the implicit `else`.  IMHO, `when` is not necessary here as the body consist of only a single expression.  So, for me the change should have been from `(if matches? s nil)` to `(if matches? s)`.  Am I missing something?